src/resources/Interfaces.ts

```diff
@@ -167,8 +167,15 @@ export interface InitialState {
   session: {
     isLoggedIn: boolean
     userInfo: UserInfo
-    lightningPayEnabled: boolean
-    boostAmount: number
+    valueSettings: {
+      lightningNetwork: {
+        lnpayEnabled: boolean
+        globalSettings: {
+          boostAmount: number
+          streamingAmount: number
+        }
```

Comment:
Future proofing comment: I think renaming lnpayEnabled to just `enabled` and globalSettings to just `settings` might make a little more sense later on because when (and of course IF) we add another network, there properties can also be `enabled` and `settings` but for a different network. `session.lightningNetwork.enabled` is distinguishable enough